packages/terra-modal-manager/src/terra-dev-site/test/modal-manager/DisclosureComponent.scss

```diff
@@ -0,0 +1,11 @@
+@import './ModalManagerCommon';
+
+:local {
+  .header {
+    @extend %common-header;
```

Comment:
Why not directly use the `common-header`  and `common-content-form` class names in the example?

Reply:
Removed this file and moved all css to a common file.